npm/mount-utils/src/index.ts

```diff
@@ -199,10 +204,12 @@ export const injectStylesBeforeElement = (
   return insertLocalCssFiles(cssFiles, document, el, options.log)
 }
 
+/**
+ * Utility function to register CT side effects and run cleanup code during the "test:before:run" Cypress hook
+ * @param optionalCallback Callback to be called before the next test runs
+ */
 export function setupHooks (optionalCallback?: Function) {
-  // Consumed by the framework "mount" libs. A user might register their own mount in the scaffolded 'commands.js'
-  // file that is imported by e2e and component support files by default. We don't want CT side effects to run when e2e
-  // testing so we early return.
+  // We don't want CT side effects to run when e2e
```

Comment:
```suggestion
  // We don't want CT side effects to run when e2e
  // testing so we early return.
```
Part of old comment got removed.